### ISSUE TITLE
[Bug] Duet malfunction

### ISSUE THREAD

**Issue by spinoza-dat** [bromite/bromite#697]
Version: `84.0.4147.132`
Arch: `arm64`
Android version: `9.0`
Device model: `ZUK Z2121`

Chrome Duet doesn't work in Home-Search-Tab Switcher Variation.
In Vanilla Chromium 83 it used to work properly, I haven't tried Vanilla Chromium 84 (don't even think the option still exists there).

To reproduce:
1. Go to `chrome://flags/`
2. Go to Chrome Duet
3. Select Home-Search-Tab Switcher Version
4. Relaunch
5. Chrome Duet will be shown as Home-Search-Share Version.

The bug started showing up on 83.0.4103.46 from what I'm seeing, since going back to 81.0.4044.127 fixes that.

(Yes the screenshot is from a previous version but the result is the same)

![Screenshot_20200516-131720_Bromite](https://user-images.githubusercontent.com/63900993/82120490-cb461100-9786-11ea-97fa-283e402904a3.png)

Edit: yes this is a copy of #580 since it wasn't solved

**Comment by csagan5** [comment]
Duplicate of #580

**Comment by csagan5** [comment]
@spinoza-dat it will never be solved here, you need to request this in the Chromium issue tracker.

**Comment by spinoza-dat** [comment]
@csagan5 wait, it works fine on Chromium. Why should I?

**Comment by csagan5** [comment]
> I haven't tried Vanilla Chromium 84 (don't even think the option still exists there).

Let's start from here. Can you reproduce the working behavior in current vanilla Chromium? And the non-working behavior in the corresponding Bromite?

**Comment by spinoza-dat** [comment]
@csagan5 proper behaviour cannot be replicated on Chromium 84, since Duet is an exclusive feature of Bromite since v84. But I can replicate a correct behaviour both on last v83 Chromium and on last v83 Bromite. Do you need log files?

**Comment by spinoza-dat** [comment]
Here are the screenshots. The version is 83.0.4103.119 for both Vanilla Chromium and Bromite.

This is the correct behaviour, on Chromium:
![Screenshot_20200820-103005_Chromium](https://user-images.githubusercontent.com/63900993/90745271-7efff500-e2d0-11ea-9865-5a33baccda5c.png)

This is the result on Bromite:
![Screenshot_20200820-102955_Bromite](https://user-images.githubusercontent.com/63900993/90745528-8fb06b00-e2d0-11ea-8ad3-33166b219065.png)

This is the selected option in chrome://flags/#enable-duet.
![Screenshot_20200820-102617_Bromite](https://user-images.githubusercontent.com/63900993/90745766-a060e100-e2d0-11ea-8dbb-e62b8036fc2b.png)

**Comment by tp0** [comment]
> since Duet is an exclusive feature of Bromite since v84.

The flag was re-added but the code that Chromium removed was not.

https://github.com/bromite/bromite/issues/667#issuecomment-668397131

While I don't understand why Chromium devs removed Duet I also don't see what is the thing that is supposed fixed here as Bromite is already on v84.

**Comment by spinoza-dat** [comment]
@tp0 there is an old unsolved duet bug on Bromite since v83. I already wrote three times what the bug is, just read above or go to #580.
On v81 duet works fine on both Chromium and Bromite.
On v83 duet is bugged on Bromite and works fine on Chromium.
On v84 duet doesn't exist on Chromium and on Bromite duet still has the v83 bug.
Chromium has nothing to do with the bug, it's a Bromite problem. Still I posted the screenshots to show the correct behaviour on Chromium
